amundsen_application/static/js/components/TableDetail/OwnerEditor/index.tsx

```diff
@@ -198,11 +206,30 @@ export class OwnerEditor extends React.Component<OwnerEditorProps, OwnerEditorSt
         <ul className='component-list'>
           {
             Object.keys(this.state.itemProps).map((key) => {
+              const owner = this.state.itemProps[key]
+              const avatarLabel = React.createElement(AvatarLabel, owner)
+
+              let listItem;
+              if (owner.link === undefined) {
+                listItem = avatarLabel
+              } else if (owner.isExternal) {
+                listItem =
+                  <a href={owner.link} target="_blank" id={`table-owners:${key}`} onClick={logClick}>
+                    { avatarLabel }
+                  </a>
+              } else {
+                listItem =
+                  <Link to={owner.link} target="_blank" id={`table-owners:${key}`} onClick={logClick}>
```

Comment:
I don't think you need the `target="_blank"` here.

Reply:
Personally me more liked behavior with target="_blank" when we open each user profile as a new browser tab, but I looked through the other links and see that this attribute used for external links only (<a/>) so yes, fixed for the sake of consistency